### PR TITLE
Banner customization

### PR DESCRIPTION
**Pull request by salinda-yasas-99**

### CODE REVIEW

File: src/main/java/com/websafe/ccmsbe/controller/CookieBannerController.java [hunk 1]
```diff
@@ -0,0 +1,27 @@
+package com.websafe.ccmsbe.controller;
+import com.websafe.ccmsbe.entity.CookieBanner;
+import com.websafe.ccmsbe.service.CookieBannerService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+
```
Comment (nimeshnuwanga) [inline src/main/java/com/websafe/ccmsbe/controller/CookieBannerController.java:7]: remove unnecessary lines

File: src/main/java/com/websafe/ccmsbe/controller/CookieBannerController.java [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+@RestController
+@RequestMapping(value="api/v1/banners")
+@CrossOrigin(value = "http://localhost:3000")
+public class CookieBannerController {
+
+    @Autowired
+    private CookieBannerService cookieBannerService;
+
```
Comment (nimeshnuwanga) [inline src/main/java/com/websafe/ccmsbe/controller/CookieBannerController.java:15]: remove unnecessary lines

File: src/main/java/com/websafe/ccmsbe/controller/CookieBannerController.java (hunk 1 above)
Comment (salinda-yasas-99) [reply src/main/java/com/websafe/ccmsbe/controller/CookieBannerController.java:7]: fixed

File: src/main/java/com/websafe/ccmsbe/controller/CookieBannerController.java (hunk 2 above)
Comment (salinda-yasas-99) [reply src/main/java/com/websafe/ccmsbe/controller/CookieBannerController.java:15]: fixed

File: src/main/java/com/websafe/ccmsbe/entity/CookieBanner.java [hunk 3]
```diff
@@ -1,9 +1,11 @@
 package com.websafe.ccmsbe.entity;
 
+import com.fasterxml.jackson.annotation.JsonBackReference;
 import jakarta.persistence.*;
 import lombok.AllArgsConstructor;
 import lombok.Data;
 import lombok.NoArgsConstructor;
+
```
Comment (nimeshnuwanga) [inline src/main/java/com/websafe/ccmsbe/entity/CookieBanner.java:8]: remove unnecessary lines

Comment (salinda-yasas-99) [reply src/main/java/com/websafe/ccmsbe/entity/CookieBanner.java:8]: fixed

**Review by nimeshnuwanga — APPROVED** [review]

**Pull request merged**